packages/design-system/src/components/section-title.tsx

```diff
@@ -197,7 +197,7 @@ export const SectionTitleLabel = forwardRef(
           // Instead, we need to render the label using a div that has position:absolute and pointer-events:none
```

Comment:
See comment it was intentionally set to inherit to inherit pointer-events none

Reply:
So your code and comments are wrong. I would check why it was intentionaly set as pointer-events:none and inherit and this is not breaking that case

Reply:
But that's disabling tooltips. And I couldn't understand what exactly need to be made from the comment.

Reply:
Having comment something has been done here with intent. You are breaking this intent but leaving old comment. The easiest way is try to find PR where it was implemented probably something would be clear